src/features/emails/utils/rendering/renderEmailHtml.spec.ts

```diff
@@ -309,6 +309,86 @@ describe('renderEmailHtml', () => {
 
     expect(output).toEqual(expected);
   });
+
+  it('sanitizes user inputs', async () => {
+    const email = mockEmail({
+      subject: '<script>alert(4)</script>',
+      content: JSON.stringify({
+        blocks: [
+          {
+            kind: 'paragraph',
+            data: {
+              // We want to invite your to[our **PARTY**](https://zetkin.org), {{target.first_name}}.
```

Comment:
Does this comment serve a purpose?

Reply:
Great question. I'm guessing it's a copy from another test. What do you say, @Herover?